pennylane/ops/qubit/parametric_ops_multi_qubit.py

```diff
@@ -261,6 +261,10 @@ class PauliRot(Operation):
         "Z": np.array([[1, 0], [0, 1]]),
     }
 
+    @classmethod
+    def _primitive_bind_call(cls, theta, pauli_word, wires=None, id=None):
+        return super()._primitive_bind_call(theta, pauli_word=pauli_word, wires=wires, id=id)
```

Comment:
The purpose of this one is to ensure `pauli_word` is static right?

Reply:
Exactly. `pauli_word` is strictly metadata, so it always needs to be a keywrod argument.